project-base/storefront/docker/entrypoint.sh

```diff
@@ -10,6 +10,9 @@ case "$1" in
     pnpm install --frozen-lockfile
     pnpm run build
     exec pnpm run start ;;
+  "dev-debug")
+    pnpm install
+    exec pnpm run dev-debug ;;
```

Comment:
I would use it directly in dev mode, so developers do not need to change things for this to work